src/pageCtrl.js

```diff
@@ -1,54 +1,79 @@
 'use strict';
-let app = angular.module('app',[]);
+/**
+ * Checks if contents of the file is in JSON format
+ */
+const isJson = (term) => {
```

Comment:
Is this supposed to be a utility function?
If so, it shouldn't be here. Also, why are you declaring functions as `const`?